applications/asset_tracker_v2/src/cloud/lwm2m_integration/lwm2m_integration.c

```diff
@@ -296,6 +296,7 @@ static int firmware_update_state_cb(uint8_t update_state)
 	case STATE_UPDATING:
 		LOG_DBG("STATE_UPDATING, result: %d", update_result);
 		cloud_wrap_evt.type = CLOUD_WRAP_EVT_FOTA_DONE;
+		lwm2m_firmware_set_update_state_cb(NULL);
```

Comment:
This needs some documentation.

1. Add here something like "Disable further callbacks from FOTA"
2. Add documentation into client libraries to indicate that NULL is a valid value.